tests/Doctrine/Migrations/Tests/Functional/FunctionalTest.php

```diff
@@ -524,7 +524,7 @@ public function testMigrationExecutedAt() : void
     }
 
     /**
-     * @return Connection|Configuration[]
+     * @return object[] [Connection, Configuration]
```

Comment:
Is this correct? Didn't see this syntax before :stuck_out_tongue:

Reply:
Yeah, looks weird, imho has no meaning in PHPStan context.

Reply:
I am not familiar with this either. I didn't make the change. It was in the first commits by @Majkl578 https://github.com/doctrine/migrations/pull/765/commits/bf980f3dc2ff16badb7fda9cb03a3b7442936e87

Reply:
I think it should be this:

```
@return Connection[]|Configuration[]
```

Reply:
Afaik it was intended to indicate a tuple consisting of Connection and Configuration, not array of those.

Reply:
phpstan didn't complain about the syntax. Should I change it back?